packages/ppx_deriving/ppx_deriving.4.3/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "whitequark <whitequark@whitequark.org>"
+authors: [ "whitequark <whitequark@whitequark.org>" ]
+license: "MIT"
+homepage: "https://github.com/whitequark/ppx_deriving"
+doc: "https://whitequark.github.io/ppx_deriving"
+bug-reports: "https://github.com/whitequark/ppx_deriving/issues"
+dev-repo: "git+https://github.com/whitequark/ppx_deriving.git"
+tags: [ "syntax" ]
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+]
+depends: [
+  "dune"     {build >= "1.0.0"}
```

Comment:
Does this really work with dune 1.0, @rgrinberg @diml? istr lots of pps fixes post 1.0 that might or might not affect this package

Reply:
I'm not sure. It depends if you want to be pessimistic or optimistic. Off the top of my head, I cannot think of any reason why dune 1.0 wouldn't work with ppx_deriving, but as you've said, there's definitely been some bugs along the way.

Reply:
Personally, I never take any chances and always put the version of Dune I'm currently using as a lower bound. Since Dune is really good at backward compatibility, upping the lower-bound is a no-brainer.

Reply:
In this case, the version that I used to develop the port was 1.6.3.